pymc/distributions/dist_math.py

```diff
@@ -115,6 +117,45 @@ def std_cdf(x):
     return 0.5 + 0.5 * pt.erf(x / pt.sqrt(2.0))
 
 
+def truncated_logcdf_from_base_rv(base_rv_op, value, lower, upper, *rv_inputs, **kwargs):
```

Comment:
I think we don't need kwargs

```suggestion
def truncated_logcdf_from_base_rv(base_rv_op, value, lower, upper, *rv_inputs):
```